src/Uno.UI/UI/Xaml/UIElement.skia.cs

```diff
@@ -288,24 +288,14 @@ internal void ArrangeVisual(Rect finalRect, Rect? clippedFrame = default)
 					throw new InvalidOperationException($"{this}: Invalid frame size {newRect}. No dimension should be NaN or negative value.");
 				}
 
-				Rect? clip;
-				if (this is Controls.ScrollViewer)
```

Comment:
Looks like this changed behavior for `ScrollViewDynamicPage` sample?

Could you try out the sample in a blank WinUI app and see the behavior? (the sample isn't currently included in WinUI SamplesApp, so easier to test in a new app)

Reply:
@ramezgerges I confirmed the new behavior is the correct one. So this is actually a bug fix 🎉